core/components/css-utilities/Border/Border.story.tsx

```diff
@@ -129,50 +123,138 @@ export const border = () => {
+            </div>
+          </div>
+          <div>
+            <div className="text-center mb-2">border border-left-0</div>
+            <div className="border border-left-0 w-30 h-30 bg-jamun-lightest d-flex align-items-center justify-content-center">
+              No left border
+            </div>
+          </div>
+        </div>
+        <div className="mb-2 color-text-subtle font-size-xs">
+          These utilities let you remove borders from specific sides of an element.
+        </div>
       </div>
-      &nbsp;
-      <div className="DocPage-codeBlock w-50 pl-10">
-        {'<span className="border border-0 bg-secondary-lightest p-8 d-inline-block mr-6">border-0</span>'}
-        <br />
-        {'<span className="border border-top-0 bg-secondary-lightest p-8 d-inline-block mr-6">top-0</span>'}
-        <br />
-        {'<span className="border border-right-0 bg-secondary-lightest p-8 d-inline-block mr-6">right-0</span>'}
-        <br />
-        {'<span className="border border-bottom-0 bg-secondary-lightest p-8 d-inline-block mr-6">bottom-0</span>'}
-        <br />
-        {'<span className="border border-left-0 bg-secondary-lightest p-8 d-inline-block">left-0</span>'}
+
+      {/* Border radius examples */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          Border Radius
+        </Heading>
+        <div className="mb-4 d-flex flex-wrap gap-5">
+          <div>
+            <div className="text-center mb-2">rounded-2-5</div>
+            <div className="border rounded-2-5 w-30 h-30 bg-stone-lightest d-flex align-items-center justify-content-center">
+              1px radius
+            </div>
+          </div>
+          <div>
+            <div className="text-center mb-2">rounded-05</div>
+            <div className="border rounded-05 w-30 h-30 bg-jal-lightest d-flex align-items-center justify-content-center">
+              2px radius
+            </div>
+          </div>
+          <div>
+            <div className="text-center mb-2">rounded-10</div>
+            <div className="border rounded-10 w-30 h-30 bg-neem-lightest d-flex align-items-center justify-content-center">
+              4px radius
+            </div>
+          </div>
+          <div>
+            <div className="text-center mb-2">rounded-20</div>
+            <div className="border rounded-20 w-30 h-30 bg-haldi-lightest d-flex align-items-center justify-content-center">
+              8px radius
+            </div>
+          </div>
+          <div>
+            <div className="text-center mb-2">rounded-full</div>
+            <div className="border rounded-full w-30 h-30 bg-jamun-lightest d-flex align-items-center justify-content-center">
+              Full radius
+            </div>
+          </div>
+        </div>
+        <div className="mb-2 color-text-subtle font-size-xs">
+          These utilities help you apply different border radius values to your elements.
+        </div>
```

Comment:
Please show example for all the existing border radius classes and not just for the few of them.

![image](https://github.com/user-attachments/assets/4d580b3e-6173-44e9-9286-cf486817f8ad)